lib/property_sets.rb

```diff
@@ -10,16 +10,19 @@
 module PropertySets
   def self.ensure_property_set_class(association, owner_class_name)
     const_name = "#{owner_class_name}#{association.to_s.singularize.camelcase}".to_sym
+
     unless Object.const_defined?(const_name)
-      property_class = Object.const_set(const_name, Class.new(ActiveRecord::Base))
-      property_class.class_eval do
+      property_class = Class.new(ActiveRecord::Base) do
```

Comment:
❤️

Reply:
I've pushed aa939fa down to my other branch (that's on top of this one because they're too hard to separate)